docs/devcontainer-setup.md

```diff
@@ -2,20 +2,50 @@
+⚠ If you are on Windows, make sure that you configure git to use linux line ending (LF) instead of windows line endings (CRLR) before checking out this repository on your local machine. Follow the [Set up git on windows](./before-you-start-on-windows.md) instructions.
 
-Once the database has been loaded run the following steps:
+Once you have the tools mentioned above installed, just open this repository in VS Code. You should see a notice in the lower left that asks if you want to open this again inside a devcontainer. Answer yes and it will spin that up. Note that the first time you run this it needs to download and ingest the database which takes 5-20 minutes. To see if the database loading has finished refer to the [Checking the docker compose logs](#checking-the-docker-compose-logs) section
 
-1. Run `yarn` in the terminal in VS code to install all JavaScript dependencies including TypeScript and webpack
-2. Run `yarn buildTsc` to build the first version of our JS artifacts
-3. run `yarn startTmuxServer`. This should split the terminal in 3 panes using tmux and run 3 components, one in each pane:
-    1. The `typescript --watch` command that recompiles TypeScript whenever you make a change to a _.ts or _.tsx file
-    2. The admin server so you can create charts in a web browser
-    3. The webpack server to build all the CSS files
+If you had the devcontainer setup running previously and get a message on the lower right asking to rebuild the devcontainer then confirm this to apply any changes to the configuration that have been made.
 
-Now navigate to http://localhost:3030/admin/charts and have a look at the admin interface. The default user account is `admin@example.com` with a password of `admin`
+Once the database has been loaded run the following steps in VS Code's terminal (i.e. the terminal running inside the devcontainer):
 
-Press `Ctrl/Cmd B + &` to kill the window and end all 3 processes when you are done
+```bash
+make up.devcontainer
+```
+
+This will run [tmux](https://github.com/tmux/tmux/wiki/Getting-Started) and create 3 panels that you can switch between with `<C-b>, n` (i.e. press `CTLR` (on PC)/`CMD` (on Mac) and `b`, then release and press `n`). Mouse support is enabled so you should be able to scroll through the panels and click them in the bottom row. A short welcome message is printed on the initial pane - if you scroll up here you will see a quick cheatsheet with various commands.
+
+Now navigate to http://localhost:3030/admin/charts in a browser and have a look at the admin interface. The default user account is `admin@example.com` with a password of `admin`
+
+To stop the admin servers, press `<C-b>, Q` in the terminal to kill the window and end all 3 processes when you are done. Close VS Code to shut down all docker containers and free up the resources of running the MySQL docker container.
+
+## Accessing Mysql
```

Comment:
Anal-retentive style comment: can we try to always write "MySQL" when in docs? 😊